python/ml4ir/base/model/relevance_model.py

```diff
@@ -635,11 +635,14 @@ def save(
         # Save individual layer weights
         self.file_io.make_directory(os.path.join(model_file, "layers"), clear_dir=True)
         for layer in self.model.layers:
-            self.file_io.save_numpy_array(
-                np_array=layer.get_weights(),
-                file_path=os.path.join(model_file, "layers", "{}.npz".format(layer.name)),
-                zip=True,
-            )
+            try:
```

Comment:
This would fail in case the `file_path` does not exist I assume. Any other cases?

Reply:
OK just saw the ErrorType (`FileNotFoundError`)..